src/gobimport/converter.py

```diff
@@ -7,20 +7,24 @@
 
 """
 import re
-
 from decimal import Decimal
+
 from gobcore.typesystem import get_value, get_gob_type_from_info
-from gobcore.model import GOBModel
 from gobcore.model.metadata import FIELD
 from gobcore.exceptions import GOBException, GOBTypeException
 from gobcore.logging.logger import logger
 
+from gobimport import gob_model
+
 
 class Converter:
 
     def __init__(self, catalog_name, entity_name, input_spec):
-        self.gob_model = GOBModel()
-        collection = self.gob_model.get_collection(catalog_name, entity_name)
+        self.gob_model = gob_model
+        try:
+            collection = self.gob_model[catalog_name]['collections'][entity_name]
+        except KeyError:
+            collection = None
```

Comment:
I don't like how this pattern is repeated over and over again. It's hard to read, compared to the old simple, very readable method call.

Reply:
The KeyError try/except can be removed after fixing the tests.